ocaml/lambda/lambda.ml

```diff
@@ -211,6 +211,8 @@ type primitive =
   | Popaque
   (* Statically-defined probes *)
   | Pprobe_is_enabled of { name: string }
+  (* Local allocations *)
```

Comment:
It's not really about local allocations. Could you think of a better comment ?

Reply:
Fixed